hyo2/soundspeed/profile/dicts.py

```diff
@@ -56,9 +56,8 @@ def first_match(cls, dct, val):
         ('Simrad', 115),
         ('OceanScience', 116),
         ('AML', 117),
-        ('SeaAndSun', 118),
-        ('HYPACK', 119),
-        ('RBR', 120),
+        ('HYPACK', 118),
```

Comment:
This approach will introduce a couple of bug in project files created with past releases: 
- The existing SeaAndSun profiles will be mislabeled as HYPACK (similarly, the existing HYPACK as RBR).
- The existing RBR profiles will crash because of missing 120 entry.

My proposed solution is to keep the current pairs, and to add the 3 proposed ones (SST-CTD, SST-CTP and SST-MEM) as 121, 122 and 123 (rather than 400, 401 and 402).  

We can add a deprecated comment for ('SeaAndSun', 118). However, no new profiles will be created with 118 because of your proposed changes to the SeaAndSun driver.